Mathlib/Algebra/FreeAlgebra.lean

```diff
@@ -573,4 +574,27 @@ theorem induction {C : FreeAlgebra R X → Prop}
   exact of_id a
 #align free_algebra.induction FreeAlgebra.induction
 
+@[simp]
+theorem adjoin_range_ι : Algebra.adjoin R (Set.range (ι R : X → FreeAlgebra R X)) = ⊤ := by
+  set S := Algebra.adjoin R (Set.range (ι R : X → FreeAlgebra R X))
+  refine top_unique fun x hx => ?_; clear hx
```

Comment:
Why do you need to clear hx here, out of curiosity?

Reply:
I just copied this from the `MvPolynomial` proof (along with the rest of the proof)